components/dashboard/src/data/projects/create-project-mutation.ts

```diff
@@ -16,26 +16,27 @@ export const useCreateProject = () => {
     const refreshProjects = useRefreshProjects();
     const { data: org } = useCurrentOrg();
 
-    return useMutation<Project, Error, CreateProjectArgs>(
-        async ({ name, slug, cloneUrl, appInstallationId }) => {
-            if (!org) {
-                throw new Error("No org currently selected");
-            }
-
-            return await getGitpodService().server.createProject({
-                name,
-                slug,
-                cloneUrl,
-                teamId: org.id,
-                appInstallationId,
-            });
-        },
-        {
-            onSuccess: (project) => {
-                if (org) {
-                    refreshProjects(org.id);
-                }
-            },
-        },
-    );
+    return useMutation<Project, Error, CreateProjectArgs>(async ({ name, slug, cloneUrl, appInstallationId }) => {
+        if (!org) {
+            throw new Error("No org currently selected");
+        }
+
+        // ensure a .git suffix
+        const normalizedCloneURL = /\.git$/.test(cloneUrl) ? cloneUrl : `${cloneUrl}.git`;
```

Comment:
Would a more JS-native approach make sense here?
```suggestion
        const normalizedCloneURL = cloneUrl.endsWith(".git") ? cloneUrl : `${cloneUrl}.git`;
```